.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Community Slack
-    url: https://t.winglang.io/slack
```

Comment:
@ainvoner i am wondering if we should also point the /slack link to go to discord, in case we have some links that we missed all around the place (even old newsletters and stuff)

Reply:
@revitalbarletz updated https://t.winglang.io/slack to point to https://discord.gg/nnbAkuKk